src/parts/AccountModal/index.tsx

```diff
@@ -32,9 +30,7 @@ import useGetAccounts from 'utils/hooks/api/use-get-accounts';
 import { shortAddress } from 'common/utils/utils';
 import { StoreType } from 'common/types/util.types';
 import { changeAddressAction } from 'common/actions/general.actions';
-import { ReactComponent as PolkadotExtensionLogoIcon } from 'assets/img/polkadot-extension-logo.svg';
-
-const POLKADOT_EXTENSION = 'https://polkadot.js.org/extension/';
+import { WalletSourceName, WALLETS } from 'utils/constants/wallets';
```

Comment:
This might be related to preference. But I think they should be defined in config rather than constants. Maybe @tomjeatt can comment on it.

Reply:
Yes, good suggestion - we have config now for the relay-chain and the vaults, I think it makes sense to have a config for supported wallets as well.